docs/3.0rc/deploy/index.mdx

```diff
@@ -12,7 +12,9 @@ search:
   boost: 2
 ---
 
-[Deployments](/3.0rc/deploy/serve-flows/) are server-side representations of flows. 
+## What is a deployment?
```

Comment:
```suggestion
## Deployments
```

Reply:
Stellar was in favor of not leading with a question. Probably good to stick with that for consistency. We should add in our style guide.